notification/test/resources/payment-credit-card.json

```diff
@@ -1,29 +1,21 @@
 {
-  "id": "123212321",
-  "version": "2",
+  "key": "testReference",
```

Comment:
I saw in extension you changed this to `YOUR_ORDER_NUMBER`. Maybe it's good to do it here as well (and in makePaymentRequest field below)?

Reply:
Sure, thanks for catching it 👍🏽

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/395/commits/2422d775cb77071b71c90af494b8d5f4060b49d9